### PR TITLE
feat(OPAL-228-Account-and-calendar-should-be-selected)

### PR DESCRIPTION
**Pull request by Isshae**
…selected by default

### CODE REVIEW

File: app/src/main/java/com/greenfox/kalendaryo/SelectCalendarActivity.java [hunk 1]
```diff
@@ -70,15 +71,20 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                    Intent i = new Intent(SelectCalendarActivity.this, ChooseAccountActivity.class);
+                    Bundle bundle = new Bundle();
+                    bundle.putParcelableArrayList("googleCalendars", (ArrayList<? extends Parcelable>) googleCalendars);
+                    i.putExtra("list", kalendar);
+                    i.putExtras(bundle);
+                    startActivity(i);
+                    finish();
+                } else {
+                    Toast.makeText(SelectCalendarActivity.this, "Please select at least one calendar to merge",
+                            Toast.LENGTH_SHORT).show();;
```
Comment (bekobarna) [inline app/src/main/java/com/greenfox/kalendaryo/SelectCalendarActivity.java]: you can use Toast.LENGTH_LONG, so the user can read it properly

File: app/src/main/java/com/greenfox/kalendaryo/adapter/AccountAdapter.java [hunk 2]
```diff
@@ -8,26 +8,32 @@
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.RadioButton;
+import android.widget.RadioGroup;
 import android.widget.TextView;
 import android.widget.Toast;
+
```
Comment (bekobarna) [inline app/src/main/java/com/greenfox/kalendaryo/adapter/AccountAdapter.java]: why the empty lines were added?

File: app/src/main/java/com/greenfox/kalendaryo/services/AccountService.java [hunk 3]
```diff
@@ -29,10 +33,14 @@ public void listAccountsFromBackend(RecyclerView recycler, boolean onFragment, I
...
                 GetAccountResponse getAccountResponse = response.body();
-                accountAdapter = new AccountAdapter(getAccountResponse.getGoogleAuths(), recycler.getContext(), false);
+                List<GoogleAuth> googleAuths = getAccountResponse.getGoogleAuths();
+
+                accountAdapter = new AccountAdapter(googleAuths, recycler.getContext(), false);
                 if (!onFragment) {
-                    accountAdapter = new AccountAdapter(getAccountResponse.getGoogleAuths(), recycler.getContext(), true);
+                    accountAdapter = new AccountAdapter(googleAuths, recycler.getContext(), true);
                     kalendar = (Kalendar) intent.getSerializableExtra("list");
+                    kalendar.setOutputGoogleAuthId(googleAuths.get(0).getEmail());
```
Comment (arnikaeva) [inline app/src/main/java/com/greenfox/kalendaryo/services/AccountService.java:42]: should refactor this, discuss it

**Review by arnikaeva — CHANGES_REQUESTED** [review]
Nice work, just minor refactoring needed, let's discuss it :)

**Review by bekobarna — APPROVED** [review]

**Review by arnikaeva — APPROVED** [review]

**Pull request merged**